### ISSUE TITLE
Demo of finetuned mot20_dla34.pth - "Skip loading parameter..."

### ISSUE THREAD

**Issue by StianHanssen** [ifzhang/FairMOT#106]
I finetuned your mot20_dla34.pth model (baseline finetuned on mot20) on my own dataset, but I run into an issue when I demo my new model on my own video.
My model was fine-tuned with:
`train.py mot --exp_id ft_my_model_mot20_dla34 --gpus 2 --batch_size 4 --load_model ../models/mot20_dla34.pth --data_cfg ../src/lib/cfg/my_model.json --num_epochs 50 --lr_step 15, 30, 45 --K 500`
I run the following command to demo:
`python demo.py mot --reid_dim 128 --conf_thres 0.4 --K 500 --input-video ../videos/out.mp4 --load_model ../exp/mot/ft_my_model_mot20_dla34/model_40.pth --output-root ../results/mot20-epoch40`
I get this warning:
```
Creating model...
loaded ../exp/mot/ft_my_model_mot20_dla34/model_40.pth, epoch 40
Skip loading parameter id.2.weight, required shapetorch.Size([128, 256, 1, 1]), loaded shapetorch.Size([512, 256, 1, 1]). If you see this, your model does not fully load the pre-trained weight. Please make sure you have correctly specified --arch xxx or set the correct --num_classes for your own dataset.
Skip loading parameter id.2.bias, required shapetorch.Size([128]), loaded shapetorch.Size([512]). If you see this, your model does not fully load the pre-trained weight. Please make sure you have correctly specified --arch xxx or set the correct --num_classes for your own dataset.
```
But I do not get the warning if I run:
`python demo.py mot --reid_dim 128 --conf_thres 0.4 --K 500 --input-video ../videos/out.mp4 --load_model ../models/mot20_dla34.pth --output-root ../results/mot20-epoch40`
It seems the model is saved differently during fine tuning, as this warning only occur after I have done fine tuning on my own dataset? And I imagine it could be related to the hyper parameters.

Any of your insight would be greatly appreciated!

**Comment by StianHanssen** [comment]
Initially I got an error when I set `--reid-dim 512`, but now it suddenly worked for some reason! So problem solved!

**Comment by NaifahNurya** [comment]
Greetings @StianHanssen  , Can you help me on how to solve this issue?? Because I got the same problem on my custom dataset. I finetuned all_hrnet_v2_w18.pth on my custom dataset. However i got this warning  when running demo.py. The code run an at the end there is any detected/tracked object.

`Skip loading parameter stage4.2.fuse_layers.3.2.0.1.running_mean, required shapetorch.Size([144]), loaded shapetorch.Size([256]). If you see this, your model does not fully load the pre-trained weight. Please make sure you have correctly specified --arch xxx or set the correct --num_classes for your own dataset`


The command which i run


`python demo.py mot --load_model ../exp/mot/all_hrnet/model_30.pth --conf_thres 0.7 --arch hrnet_18 --input-video ../videos/my-video.mp4 --output-root ../results/`


This is my logfile.

![image](https://user-images.githubusercontent.com/41193844/85188320-acd0b980-b2e0-11ea-8025-e44db0aeaa7e.png)
